src/lib/hooks/useResize.ts

```diff
@@ -0,0 +1,5 @@
+import { useResizeDetector } from 'react-resize-detector';
+
+export const useResize: typeof useResizeDetector = (props) => {
+	return useResizeDetector(props);
+};
```

Comment:
Nice! I think a slightly cleaner API would look something like...
```js
const { ref, size, onResize } = useResize()

return <div ref={ref}></div>
```
(This would be in a component using `useResize`).

That way, you wouldn't have to store any state in the component and the hook does the whole thing. Very minor improvement though so not necessary for approval!